eZ/Publish/Core/Persistence/Cache/Adapter/TransactionalInMemoryCacheAdapter.php

```diff
@@ -155,7 +155,7 @@ public function invalidateTags(array $tags)
     /**
      * {@inheritdoc}
      */
-    public function clear()
+    public function clear(string $prefix = '')
```

Comment:
@emodric `clear` is implemented from the interface [`\Psr\Cache\CacheItemPoolInterface::clear`](https://github.com/php-fig/cache/blob/master/src/CacheItemPoolInterface.php#L77).
I don't see any changes there, so this change is inconsistent with the interface. 

I see that `\Symfony\Component\Cache\Adapter\AdapterInterface` has this method added in Symfony 4.4, but since the inconsistency I'm not sure if we should change it right now. At least we need to wait until 4.4. is released.

Reply:
Removed the commit :+1: